src/main/scala/io/iohk/ethereum/vm/DataWord.scala

```diff
@@ -78,9 +72,14 @@ class DataWord private (private val n: BigInt) extends Ordered[DataWord] {
 
   def /(that: DataWord): DataWord = DataWord(this.n / that.n)
 
+  def **(that: DataWord): DataWord = DataWord(this.n.modPow(that.n, Modulus))
+
   def compare(that: DataWord): Int = this.n.compare(that.n)
 
-  def intValue: Int = n.intValue
+  /**
+    * @return an Int with MSB=0, thus a value in range [0, Int.MaxValue]
+    */
+  def intValue: Int = n.intValue & Int.MaxValue
```

Comment:
is this necessary if we have `require(n >= 0 && n < Modulus, s"Invalid word value: $n")`
maybe just def intValue: Int = n.intValue
in case of integer overflow it will return unexpected values for example:
```
scala> BigInt("3000000000").intValue
res11: Int = -1294967296
```
```
scala> (Int.MaxValue & BigInt("3000000000").intValue)
res12: Int = 852516352
```
but this case is covered by `< Modulus` so do we need this `& Int.MaxValue` ?